daemon/container_operations_windows.go

```diff
@@ -35,6 +43,62 @@ func detachMounted(path string) error {
 	return nil
 }
 
+func (daemon *Daemon) setupSecretDir(c *container.Container) (setupErr error) {
+	if len(c.SecretReferences) == 0 {
+		return nil
+	}
+
+	localMountPath := c.SecretMountPath()
+	logrus.Debugf("secrets: setting up secret dir: %s", localMountPath)
+
+	defer func() {
+		if setupErr != nil {
+			if err := os.RemoveAll(localMountPath); err != nil {
+				log.Errorf("error cleaning up secret mount: %s", err)
+			}
+		}
+	}()
+
+	// create secrets root
+	system.MkdirAll(localMountPath, 0)
```

Comment:
I don't think there's a lot of choice but to follow the root, the same as for everything else. Otherwise, if there's a static path, how would two daemons run side-by-side (such as they do in CI conditions, and some unusual other use-cases I've seen). Sure, you could put it under `TEMP` in a folder unique to that daemon instance, but does an external entity need to be able to determine the path an populate files? If so, that would need to be exposed via `/INFO` for example.

That ACL thing is definitely concerning - would appreciate verification it doesn't regress it.